R/animint.R

```diff
@@ -991,7 +1047,11 @@ servr::httd("', normalizePath( out.dir,winslash="/" ), '")')
 #' @return True/False value
 #' @export
 is.rgb <- function(x){
-  grepl("NULL", x) | (grepl("#", x) & nchar(x)==7)
+  if(is.null(x)) {
+    TRUE
+  } else {
+    (grepl("#", x) & nchar(x)==7)
```

Comment:
Maybe we could change this line to `grepl("^#[0-9A-Za-z]{6}$", x)` ?

cc @tdhock